pkgs/tools/security/sha1collisiondetection/default.nix

```diff
@@ -0,0 +1,39 @@
+{ stdenv, fetchFromGitHub, libtool }:
+
+stdenv.mkDerivation  rec {
+  pname = "sha1collisiondetection";
+  version = "git-20170221";
```

Comment:
The idea is that it should generate _exactly_ the same hashes for the 'real' artefacts, but will actually generate another one for manipulated artefacts. The possibility of false positives can be neglected as the probability is smaller than 2^-90.